test/ui/history_apply.multiout

```diff
@@ -0,0 +1,86 @@
+++++++++++++++++++++++++++++++++++++++++++++++++++
+|In f_invoke(x, y, z) at ui.jl:70
+| 69  function f_invoke(x, y, z)
+|>70      return g_invoke(x..., y, z...)
+| 71  end
+|
+|About to run: (tuple)(3)
```

Comment:
Ideally, this would be skipped but not sure how to detect this "splat setup" right now.